sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobServiceImpl.java

```diff
@@ -445,23 +407,23 @@ BlobDownloadResponse downloadWithRestResponse(String containerName,
      * @param cpkInfo              Additional parameters for the operation.
      * @param callback             Callback that receives the response.
      */
-    void downloadWithRestResponse(String containerName,
-                                  String blobName,
-                                  String snapshot,
-                                  Integer timeout,
-                                  String range,
-                                  String leaseId,
-                                  Boolean rangeGetContentMD5,
-                                  Boolean rangeGetContentCRC64,
-                                  OffsetDateTime ifModifiedSince,
-                                  OffsetDateTime ifUnmodifiedSince,
-                                  String ifMatch,
-                                  String ifNoneMatch,
-                                  String version,
-                                  String requestId,
-                                  CpkInfo cpkInfo,
-                                  CancellationToken cancellationToken,
-                                  Callback<BlobDownloadResponse> callback) {
+    void rawDownload(String containerName,
```

Comment:
If there's no other type of download, do we really need to name this `rawDownload()`?

Reply:
yes, this is what we agreed across iOS and android, `rawDownload` for the protocol layer and `download` for the managed download.